stories/SQFormDialogStepper.stories.js

```diff
@@ -240,6 +242,17 @@ export const SQDialogStepperWithValidation = () => {
             lastName: Yup.string().required('Required')
           }}
         >
```

Comment:
For the stories we want the examples to be as realistic/good as the code a consumer would write.  We should push the textButton into a section header like we have done in our app.